index.js

```diff
@@ -265,6 +265,14 @@ defineLazyProperty(apps, 'firefox', () => detectPlatformBinary({
 	wsl: '/mnt/c/Program Files/Mozilla Firefox/firefox.exe'
 }));
 
+defineLazyProperty(apps, 'edge', () => detectPlatformBinary({
+	darwin: 'microsoft edge',
+	win32: 'msedge',
+	linux: 'microsoft-edge'
+}, {
+	wsl: '/mnt/c/Program Files (x86)/Microsoft/Edge/Application/msedge.exe',
```

Comment:
You need to add the 64-bit path too.

Reply:
tested on windows with 64-bit edge, but looks like they still installed under x86 path only.
no idea why they didn't align this design with chrome.